popup.js

```diff
@@ -27,6 +27,7 @@ chrome.tabs.query({ currentWindow: true, active: true }, function (tabs) {
     // Check off checkbox if currently reloading
     if(items[tab.id] && items[tab.id].active){
         active.checked = true;
+        activeLabel.innerHTML = 'ON';
```

Comment:
```suggestion
        activeLabel.innerHTML = 'ON';
        settingsContainer.classList.add('disabled');
```

Reply:
Ah yes, I forgot to add that in after the popup loads again while it's running. It's fixed but I didn't make a separate commit for it.